scenarios/crowdsecurity/http-bf-wordpress_bf.md

```diff
@@ -1,4 +1,4 @@
 Detects bruteforce on wordpress login page 'wp-login.php'.
 
-leakspeed of 10s, capacity of 5
```

Comment:
You need to keep this scenario with its defaults values.

Reply:
reverted in d42f77ca6e0ae9d4d537818eb50d60f72f17c83a